conf/yarn-site.xml.template

```diff
@@ -66,5 +66,9 @@
         <name>yarn.nodemanager.pmem-check-enabled</name>
 	<value>false</value>
     </property>
-
+    <property>
```

Comment:
Are you sure we need this ratio to avoid the killing? I'm under the impression that disabling the check is sufficient.